test/e2e/inference_with_adapters_test.go

```diff
@@ -104,6 +123,39 @@ func validateAdapterAdded(workspaceObj *kaitov1alpha1.Workspace, deploymentName
 	})
 }
 
+func validateNoInitContainers(workspaceObj *kaitov1alpha1.Workspace, expectedInitContainers []corev1.Container) {
```

Comment:
Why is the method named "NoInitContainers"?

Reply:
To validate the previous InitContainers has been deleted